config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=19.2
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=19.2.0.2
+VERSION_LONG=19.2.0.3
```

Comment:
Given your decision on p5T066-2YK-p2#comment-11376 I'm understanding that you will release 19.2.0.3 (the one you submitted on Friday and I'm guessing was since approved) to the store and follow up by a 19.2.1 hotfix — as opposed to discarding 19.2.0.3 and submit a 19.2.0.4 with the fix instead, right?

In that case this seems like the right version and this will still be the last release/19.2->trunk PR 👍 But just wanted to validate that point anyway

Reply:
Correct.

Merging.

Thank you for the review 🙇